chef-config/lib/chef-config/exceptions.rb

```diff
@@ -20,7 +20,7 @@
 
 module ChefConfig
 
-  class InvalidPath < StandardError
-  end
+  class ConfigurationError < ArgumentError; end
```

Comment:
@coderanger [this should take care of it](https://github.com/chef/chef/blob/6a4f414666b3688a3dfa94b131f83235f4d2fa64/lib/chef/exceptions.rb#L28)